index.js

```diff
@@ -77,41 +77,63 @@ function getScopedCid(scope, apiKey, callback) {
     if (!token || isStatusToken(token)) {
       persistToken(TokenStatus.RETRIEVING, TIMEOUT);
     }
-    fetchCid(GOOGLE_API_URL + apiKey, scope, token, cb)
+    fetchCid(apiKey, scope, token, cb)
   });
 }
 
 /**
  * Fetches CID remotely
  *
- * @param {string} url
+ * @param {string} apiKey
  * @param {string} scope
  * @param {string=} token
```

Comment:
can you change this to string|undefined

Reply:
actually in this case it should string|null since `getCookie` explicitly returns `null`

Reply:
changed to `{?string}`